module/VuFindSearch/src/VuFindSearch/Backend/SRU/Connector.php

```diff
@@ -22,7 +22,7 @@
  *
  * @category VuFind
  * @package  SRU
- * @author   Andrew S. Nagy <vufind-tech@lists.sourceforge.net>
+ * @author   Maccabee Levine <msl321@lehigh.edu>
```

Comment:
Since this is an existing file, I think it's best to leave Andrew as an original author and just add yourself.

```suggestion
 * @author   Andrew S. Nagy <vufind-tech@lists.sourceforge.net>
 * @author   Maccabee Levine <msl321@lehigh.edu>
```